### DISCUSSION TITLE
posts/r5/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#35] [Comments]
# posts/r5/

大一《政治经济学》课程老师提到了 R 语言，于是开始慢慢接触。2019 年 11 月从学校图书馆借了书开始看。后面自己也买了一些：《R 语言实战》、《R 语言之书》、《ggplot2：数据分析与图形艺术》……
[&hellip;] 现在仍记得，大一晚自习，更换镜像后安装 R 包，看到快速闪过的进度条窗口和控制台输出时的喜悦，那时还是 R 3.6.1。
[&hellip;] 对我来说，真正走上 R  &hellip;

https://shitao5.org/posts/r5/

**Comment by yihui** [comment]
政治经济学老师居然知道 R，这老师也真是个人才啊。想当年，我上（马克思主义）政治经济学时，只觉得那个唾沫横飞的老师讲起课来实在是无聊透顶，完全是浪费我们的青春时光。

**Reply by Shitao5** [reply]
政治经济学老师也是经济系的系主任，为了我们的就业真是操碎了心，推荐过 R、Python 和 Go……现在也是要求上马克思主义政治经济学，内容无聊透顶，尤其里边讲劳动价值论又让我困惑万分。好在我们的老师没有唾沫横飞，他让我们去调研水果批发市场，让我们学点 R 啥的。现在回想都是缘分，要是当时他没提 R，我现在还不知道在哪里，大概率也不会有这个博客，遇见好多人。
